stdlib/public/SDK/XCTest/XCTestCaseAdditions.mm

```diff
@@ -152,12 +150,11 @@ - (NSString *)className
                    @"reason": exception.reason,
                    };
     }
-
     @catch (...) {
         result = @{
                    @"type": @"unknown",
                    };
     }
 
-    return [result retain];
+    return result;
```

Comment:
Oops, no, that was a bug. It was necessary when we were using `@_silgen_name` to get at the function, but it's just been leaking since I took that out. :-(